### PR TITLE
[TS-167] added GoDoc to web packet

### PR DESCRIPTION
**Pull request by jenchik**
Suggest change to echo.Logger instead yaga/logger.Logger

### CODE REVIEW

**Review by kpeo — DISMISSED** [review]

File: web/web.go [hunk 1]
```diff
@@ -55,15 +58,18 @@ func New(opts Options) *echo.Echo {
 	return e
 }
 
+// Start starts an HTTP server with custom address.
```
Comment (pcherednichenko) [inline web/web.go]: `// StartServer start HTTP server with custom address.`
the name must begin with the name of the function by the standards of GoDoc

File: web/web.go [hunk 2]
```diff
@@ -20,19 +20,22 @@ const (
 	defaultBind = ":8080"
 )
 
+// Options contains a parameters for new Echo instance.
 type Options struct {
+	// TODO suggest change to echo.Logger
```
Comment (pcherednichenko) [inline web/web.go:25]: why TODO? why not leave our Logger?

Comment (im-kulikov) [reply web/web.go:25]: what's the point?

File: web/web.go (hunk 1 above)
Comment (im-kulikov) [reply web/web.go]: but.. why not `StartServer HTTP with custom address`?

**Review by pcherednichenko — APPROVED** [review]

File: web/web.go [hunk 3]
```diff
@@ -20,19 +20,22 @@ const (
...
 	Logger logger.Logger
 	Error  errors.Logic
 	Debug  bool
 }
 
+// New creates an instance of Echo.
 func New(opts Options) *echo.Echo {
-	// Enabling raven:
+	// enabling raven
```
Comment (im-kulikov) [inline web/web.go:33]: mb `enable`, what you think about?

**Review by im-kulikov — APPROVED** [review]
SGTM

**Review by kpeo — APPROVED** [review]

**Pull request merged**